tests/test_typeinfo.py

```diff
@@ -44,29 +48,69 @@ async def test_fetch_async(aconn, name, status):
     assert info.array_oid == psycopg.adapters.types["text"].array_oid
 
 
-@pytest.mark.parametrize("name", ["nosuch", sql.Identifier("nosuch")])
-@pytest.mark.parametrize("status", ["IDLE", "INTRANS"])
-def test_fetch_not_found(conn, name, status):
+_name = pytest.mark.parametrize("name", ["nosuch", sql.Identifier("nosuch")])
+_status = pytest.mark.parametrize("status", ["IDLE", "INTRANS"])
+_has_to_ragtype = pytest.mark.parametrize("has_to_regtype", [True, False])
```

Comment:
I wouldn't add this parametrization: it's not an user input. What I would do is maybe to move the test for no exception in `Transaction.__exit__` separate from the others, and to skip it on databases that are known to not pass that test.

You can easily achieve it by using:
```diff
diff --git a/tests/fix_crdb.py b/tests/fix_crdb.py
index ad85698e..32ec457b 100644
--- a/tests/fix_crdb.py
+++ b/tests/fix_crdb.py
@@ -122,6 +122,7 @@ _crdb_reasons = {
     "server-side cursor": 41412,
     "severity_nonlocalized": 81794,
     "stored procedure": 1751,
+    "to_regtype": None,
 }
 
 _crdb_reason_version = {
@@ -130,4 +131,5 @@ _crdb_reason_version = {
     "cancel": "skip < 22",
     "server-side cursor": "skip < 22.1.3",
     "severity_nonlocalized": "skip < 22.1.3",
+    "to_regtype": "skip < 22.2",
 }
```
and decorate the test with `@pytest.mark.crdb_skip("to_regtype")`